### PR TITLE
Auto-close copy notification after 3 seconds + configuration support

### PR DESCRIPTION
**Pull request by cheenamalhotra**
Addresses #24009 by auto-closing Copy notification in 3 seconds.

auto close

Additionally added support to disable notifications:
image

New behavior:
disable notifications

### CODE REVIEW

**Review by alanrenmsft — APPROVED** [review]

File: src/sql/platform/query/common/query.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ export interface IQueryEditorConfiguration {
 		readonly inMemoryDataProcessingThreshold: number;
 		readonly openAfterSave: boolean;
 		readonly showActionBar: boolean;
+		readonly dataCopyNotifications: boolean;
```
Comment (alanrenmsft) [inline src/sql/platform/query/common/query.ts:35]: showCopyCompletedNotification

**Review by alanrenmsft — APPROVED** [review]
just rename it more accurately and rest looks good to me.

File: src/sql/workbench/contrib/query/browser/query.contribution.ts [hunk 2]
```diff
@@ -303,9 +303,9 @@ const queryEditorConfiguration: IConfigurationNode = {
 			'description': localize('queryEditor.results.copyRemoveNewLine', "Configuration options for copying multi-line results from the Results View"),
 			'default': true
 		},
-		'queryEditor.results.dataCopyNotifications': {
+		'queryEditor.results.showCopyCompletedNotification': {
 			'type': 'boolean',
-			'description': localize('queryEditor.results.copyNotifications', "Enables notifications for data copy operations."),
+			'description': localize('queryEditor.results.showCopyCompletedNotification', "Enables notifications for data copy operations."),
```
Comment (alanrenmsft) [inline src/sql/workbench/contrib/query/browser/query.contribution.ts:308]: and the description...

Comment (cheenamalhotra) [reply src/sql/workbench/contrib/query/browser/query.contribution.ts:308]:
```suggestion
			'description': localize('queryEditor.results.showCopyCompletedNotification', "Enables showing notifications for copy complete operations."),
```

**Review by Charles-Gagnon — APPROVED** [review]

**Pull request merged**